pagopa/io-app
# chore: [IOBP-589] Adapted new payment flow with new swagger definitions

## Short description
This PR adapts the new API definitions changing some attributes naming

## List of changes proposed in this pull request
- Updated package.json with the new API definitions endpoint
- Renamed some attributes

## How to test
- Checkout https://github.com/pagopa/io-dev-api-server/pull/353 or test in UAT env
- You should be able to onboard a new payment method and pay without any problems with the new onboarding and payment flow from developer playground screens.
